spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/sql/DatabaseType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.cloud.autoconfigure.sql;
+
+/**
+ * @author Warren Zhu
+ */
+public enum DatabaseType {
+    SQLSERVER("com.microsoft.sqlserver.jdbc.SQLServerDriver", "jdbc:sqlserver://%s:1433;database=%s;" +
```

Comment:
Is it easy to be broken and hard to troubleshoot if we use literal text for class name?

Reply:
Class literal couldn't be used here since class might be not in the classpath.